src/Stratis.CirrusPegD/Program.cs

```diff
@@ -27,6 +29,20 @@ class Program
         private const string MainchainArgument = "-mainchain";
         private const string SidechainArgument = "-sidechain";
 
+        private static readonly Dictionary<NetworkType, Func<Network>> SidechainNetworks = new Dictionary<NetworkType, Func<Network>>
+        {
+            { NetworkType.Mainnet, FederatedPegNetwork.NetworksSelector.Mainnet },
```

Comment:
Why are we not using `CirrusNetwork`?

Reply:
'CirrusNetwork' will be the next version of Cirrus. At the moment we're still using all of the old 'FederatedPegNetwork' classes